.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "configurations": [
+        {
+            "args": [],
+            "env": {},
+            "host": "127.0.0.1",
+            "mode": "debug",
+            "name": "Launch",
+            "port": 2345,
+            "program": "${fileDirname}",
+            "remotePath": "",
+            "request": "launch",
+            "showLog": true,
+            "type": "go"
+        }
+    ],
+    "version": "0.2.0"
+}
```

Comment:
I'm not familiar with VisualStudio, but is this a file we can put in .gitignore?

Reply:
Yeah, ignoring that whole directory will be fine.
I went with [the output from gitignore.io](https://www.gitignore.io/api/go,vim,visualstudiocode) and added `visualstudiocode` to the two existing choices (`go`,`vim`) but I don't know why it would explicitly include some of those JSON files.
Just take the 4 VSCode lines that start with `!` out of `.gitignore` and scrap that whole `launch.json` file.